runtime/Cargo.toml

```diff
@@ -11,9 +11,8 @@ repository = 'https://github.com/DataHighway-DHX/node'
 targets = ['x86_64-unknown-linux-gnu']
 
```

Comment:
please make all inline, like in substrate-parachain-template, and in our standalone code, i.e. https://github.com/substrate-developer-hub/substrate-parachain-template/blob/master/runtime/Cargo.toml#L24

Reply:
added to https://github.com/DataHighway-DHX/node/issues/231